app/components/spotlight/analytics/aggregation_component.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Spotlight
+  module Analytics
+    # Display Analtyics aggregations as table
+    class AggregationComponent < ViewComponent::Base
```

Comment:
Could you add a test for this component?

Reply:
done